chcemvediet/apps/__init__.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+import django.db.backends.sqlite3.base
+from django.apps import AppConfig
+from django.db.backends import utils as backend_utils
+
+
+class ChcemvedietConfig(AppConfig):
+    name = u'chcemvediet.apps'
+
+    def ready(self):
+        django.db.backends.sqlite3.base._sqlite_format_dtdelta = self._sqlite_format_dtdelta
+
+    # fixed in django 3.1rc1
+    def _sqlite_format_dtdelta(self, dt, conn, days, secs, usecs):
+        try:
+            dt = backend_utils.typecast_timestamp(dt)
+            delta = datetime.timedelta(int(days), int(secs), int(usecs))
+            if conn.strip() == u'+':
+                dt = dt + delta
+            else:
+                dt = dt - delta
+        except (ValueError, TypeError):
+            return None
+        return str(dt).strip(u'+00:00')
```

Comment:
@martinmacko47 Mozem pouzit tento jeden konkretny timezone alebo to mam spravit viac vseobecne?

Reply:
Ano, staci fixne `+00:00`, kedz vzdy pridava iba `+00:00`.